stash/vitest.config.ts

```diff
@@ -9,14 +9,16 @@ export default defineConfig({
     globalTeardown: 'test/globalTeardown.ts',
     env: {
       APP_ENV: 'rollup-dev',
-      MANGATA_NODE_URL: 'wss://collator-01-ws-rollup-frontend.gasp.xyz',
+      MANGATA_NODE_URL: 'wss://collator-01-ws-rollup-holesky.gasp.xyz',
       REDIS_HOST: 'localhost',
       REDIS_PORT: '6380',
       REDIS_PASS: '',
       TIMESERIES_HOST: 'localhost',
       TIMESERIES_PORT: '6379',
       TIMESERIES_PASS: '',
-      COINGECKO_API_KEY: process.env.COINGECKO_API_KEY
+      COINGECKO_API_KEY: process.env.COINGECKO_API_KEY,
+      CAPTCHA_SITEKEY: '10000000-ffff-ffff-ffff-000000000001',
```

Comment:
These keys are test ones publicly available on hcaptcha website [here](https://docs.hcaptcha.com/#test-key-set-publisher-or-pro-account) but we can move them to sops if we need. @goncer opinion? stay or move to sops?